action.yml

```diff
@@ -54,14 +54,18 @@ runs:
           rm tailscale.tgz
           TSPATH=/tmp/tailscale_${VERSION}_amd64
           sudo mv "${TSPATH}/tailscale" "${TSPATH}/tailscaled" /usr/bin
-      - name: Run Tailscale
+      - name: Start Tailscale Daemon
+        shell: bash
+        run: |
+          sudo tailscaled 2>~/tailscaled.log &
+          sudo tailscale status
```

Comment:
it fails because this returns a non-zero exit code

Reply:
when `Logged out`

Reply:
fixed